tests/unit/src/Internal/Payment/State/ProcessedStateTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Class ProcessedStateTestTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests\Internal\Payment\State;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WC_Helper_Intention;
+use WCPay\Internal\Payment\PaymentContext;
+use WCPay\Internal\Payment\State\CompletedState;
+use WCPay\Internal\Payment\State\ProcessedState;
+use WCPay\Internal\Payment\State\StateFactory;
+use WCPay\Internal\Service\OrderService;
+use WCPAY_UnitTestCase;
+
+/**
+ * Tests for the initial payment state.
```

Comment:
Outdated comment.

Reply:
Comment is updated now: https://github.com/Automattic/woocommerce-payments/pull/7471/commits/48da0fc92461c2e53787cf45c2e79dcf0fa5bf8d